### DISCUSSION TITLE
"fasterMovement" >>> enhancement

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#67] [Ideas]
Being able to define different speeds per dimension.

Example:
- Overworld: Speed 1
- Nether: Speed 3

**Comment by 1024-byteeeee** [comment]
Not considering further expansion for this feature.

**Comment by ghost** [comment]
A bit sad, it would be a feature I would make a use of.

(The truth is, that was my original idea behind it, while I asked you to expand the rule to the worlds. But it was a miss communication in the past which leads to the rule of now. But it's fine 😊)
